README.md

```diff
@@ -1,51 +1,33 @@
-# Loki Operator for K8s
+# Loki Charmed Operator for K8s
 
 [![Test Suite](https://github.com/canonical/loki-k8s-operator/actions/workflows/run_tests.yaml/badge.svg)](https://github.com/canonical/loki-k8s-operator/actions/workflows/run_tests.yaml)
 
 ## Description
 
-The [Loki](https://grafana.com/oss/loki/) operator provides an open-source fully featured logging stack. This repository contains a Juju Charm for deploying Grafana Loki on Kubernetes clusters.
+[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. The Loki charmed operator deploys Loki in Kubernetes using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
 
 
-## Usage
+## Quick start
 
-Create a Juju model for your operators, say "lma"
+Create a Juju model named `observability`. This also sets the new model as the current one in the Juju command-line interface.
 
 ```bash
-    juju add-model lma
+juju add-model observability
 ```
 
-The Loki Operator may be deployed using the Juju command line
+Deploy the Loki Operator from Charmhub to the current model using the Juju command-line interface:
 
 ```bash
-    juju deploy loki-k8s
+juju deploy loki-k8s
 ```
 
-If required, you can remove the deployment completely:
+If required, you can remove the deployment completely or the whole model:
```

Comment:
This did not turn out great. I'd much prefer if the "or the whole model" is placed between them to make it obvious which one is which.